source/core/GenericFunctions.cpp

```diff
@@ -2650,6 +2650,16 @@ VIREO_FUNCTION_SIGNATUREV(MergeErrors, MergeErrorsParamBlock)
 }
 
 //------------------------------------------------------------
+// We need this operation on a single vireo instruction so boolean latched buttons work correctly
+// from(0), to(1), fromNewValue(2)
```

Comment:
nitpick --- fromNewValue is a bit confusing to read. I would do "src", "dest", "newValueForSrc"---  nitpick.  I am ok to submit as is.